desktop/analytics/partner.js

```diff
@@ -22,4 +22,20 @@
       partner_slug: $(e.currentTarget).data('partner-slug')
     })
   })
+
+  analyticsHooks.on('renderedPartner2ContactInfo', function (options) {
```

Comment:
What's the need for the event here? Can't you just define the below click handlers/selectors as-is?